docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -85,6 +83,13 @@ Preparing For a Release
 
    This uses SSH agent forwarding so that you can push changes to GitHub using the keys from your workstation.
 
+   Add your SSH key to the ``sshd`` agent.
+
+   .. prompt:: bash $
+
+      [ -e "${SSH_AUTH_SOCK}" ] || eval $(ssh-agent)
+      ssh-add $HOME/.ssh/id_rsa
```

Comment:
Do this only if the file exists, and mention explicitly that it should be changed to a key which is linked to your GitHub account.